dianna/__init__.py

```diff
@@ -71,7 +71,7 @@ def explain_image(model_or_function, input_image, method, labels, **kwargs):
         kwargs: These keyword parameters are passed on
 
     Returns:
-        One heatmap (2D array) per class.
+        An array (np.ndarray) containing the heat maps for each class.
```

Comment:
Nice and thorough!
It would be even better if we could have type hints in the function signature.
We once decided (about a year ago) to start using type hints. We decided back then to not change the complete code at once, but only update code wherever we are working on it already. Now that you already created such thorough docstrings including input and output types, it shouldn't be such a big task to add those to the function signature as well:
def explain_image(model_or_function:Union(str, Callable), input_image:ndarray, method:string, labels:Iterable[int], **kwargs) -> ndarray:
If that's too much effort or is too troublesome in certain cases, only the output types would already help.
Let me know if you disagree to include this in this PR.

Reply:
Good idea. I added types wherever possible. It would be nice if `mypy` is also added to the workflows to check the typing. see issue https://github.com/dianna-ai/dianna/issues/85